.github/kokoro/steps/vtr-test.sh

```diff
@@ -69,7 +69,8 @@ find vtr_flow/benchmarks/titan_blif/ -type f -not -name 'README.*' -delete
 find . -type f -regex ".*\.tar\.\(gz\|xz\)" -delete
 
 # Make sure working directory doesn't exceed disk space limit!
-echo "Working directory size: $(du -sh)"
+cd vtr_flow/tasks/
```

Comment:
can merge this in as commented out code, and say that can be used to see breakdown of disk space